File: Toy_Java/src/View/Command.java
package View;

/**
 * Created by vladc on 08.11.2016.
 */
public abstract class Command {
    private String key;
    private String description;

    public Command(String key, String description) {
        this.key = key;
        this.description = description;
    }

    public abstract void execute();

    public String getKey() {
        return key;
    }

    public String getDescription() {
        return description;
    }
}

File: Toy-Language-Interpreter/View/Command.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Toy_Language_Interpreter.View
{
    abstract class Command
    {
        private string key;
        private string description;

        public Command(string key, string description)
        {
            this.key = key;
            this.description = description;
        }

        public abstract void Execute();

        public string GetKey()
        {
            return key;
        }

        public string GetDescription()
        {
            return description;
        }
    }
}
